src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -459,12 +471,16 @@ void GNAGraphCompiler::finalizeConvolution1DPrimitive(InferenceEngine::CNNLayerP
 
     connectOutput(layer, ptr_outputs, num_data_bytes_out);
 
+    // Transpose H with W or C with HW
```

Comment:
@elilobanova Actually, yes. All IRs where the methodology is changed have either C either H equal to 1. That's why both variants are working. 

When all dimensions H, W and C > 1 then we will go to another function finalizeConvolution2DPrimitive, not finalizeConvolution1DPrimitive.

The only possible problem can be when convolution is satisfy the condition of isMappableFrom2DTo1D function and H became 1. Maybe there is a sense to check a number of channels here because this case is unsupported now. I will think about it.